plat/kvm/Config.uk

```diff
@@ -15,7 +15,7 @@ menuconfig PLAT_KVM
 if (PLAT_KVM)
 
 choice
-	prompt "Booting environment"
```

Comment:
The reason why I named it `Booting environment` is because IMO UEFI is not exactly a boot protocol. But, alas, the term is vague anyway. :thinking:.